README.adoc

```diff
@@ -1,138 +0,0 @@
-= Couchbase Documentation Site
```

Comment:
unsure why this is deleted, won't that just cause merge issues if it gets updated in prod?

Reply:
Ah yeah, I got a bit excited copying old sandbox approach, this should probably stay 👍 .